arangosh/Benchmark/BenchFeature.cpp

```diff
@@ -154,14 +154,14 @@ void BenchFeature::collectOptions(std::shared_ptr<ProgramOptions> options) {
       "--runs", "run test n times (and calculate statistics based on median)",
       new UInt64Parameter(&_runs));
 
-  options->addOption("--progress", "show progress",
+  options->addOption("--progress", "intermediate logging on every 20th request",
```

Comment:
"on every 20th request" is not correct.
It is actually logging every n requests, where `n` depends on the value of `--requests`.
We should probably change it to "log intermediate progress"